ui/src/app/pages/dashboard/dashboard.component.ts

```diff
@@ -97,8 +97,11 @@ export class DashboardComponent implements OnInit {
 
   getLeaderboard() {
     return (this.globalLeaderboardSelected)
-      ? this.leaderboardService.GetGlobalLeaderboard()
-      : this.leaderboardService.GetClassLeaderboard();
+      ? this.leaderboardService.GetGlobalLeaderboard().sort((a, b) => {
```

Comment:
Could be more concise by just sorting the ternary result:

```
return (
  this.globalLeaderboardSelected 
    ? this.leaderboardService.GetGlobalLeaderboard() 
    : this.leaderboardService.GetClassLeaderboard()
  ).sort((a, b) => b.score - a.score);

Reply:
Updated logic. To ternary expression.